treeherder/etl/perf.py

```diff
@@ -307,23 +307,31 @@ def _load_perf_datum(job: Job, perf_datum: dict):
                 defaults={"value": value[0], "application_version": application_version},
             )
 
-            if _test_should_gather_replicates_based_on(
-                job.repository, suite["name"], subtest.get("replicates", [])
-            ):
-                try:
-                    # Add the replicates to the PerformanceDatumReplicate table, and
-                    # catch and ignore any exceptions that are produced here so we don't
-                    # impact the standard workflow
-                    PerformanceDatumReplicate.objects.bulk_create(
-                        [
-                            PerformanceDatumReplicate(
-                                value=replicate, performance_datum=subtest_datum
-                            )
-                            for replicate in subtest["replicates"]
-                        ]
-                    )
-                except Exception as e:
-                    logger.info(f"Failed to ingest replicates for datum {subtest_datum}: {e}")
+            replicates = subtest.get("replicates", [])
+            if _test_should_gather_replicates_based_on(job.repository, suite["name"], replicates):
+                existing_replicates = set(
+                    PerformanceDatumReplicate.objects.filter(
+                        performance_datum=subtest_datum
+                    ).values_list("value", flat=True)
```

Comment:
I'm guessing this is happening because of duplicate ingestion tasks (log, and json). I think we should find a way to default to using the JSON if they exist, and ignore the data we find in the logs. Maybe we could have a list of tests that we start with for testing this out? I'm thinking we could start with these tasks since the data they produce is not useful so any failures won't be problematic: https://treeherder.mozilla.org/jobs?repo=autoland&searchStr=regress&revision=6bd2ea6b9711dc7739d8ee7754b9330b11d0719d&selectedTaskRun=K87CGE6IT1GHl6wD4Skbyw.0